README.md

```diff
@@ -339,6 +361,13 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+Close the Connection
+--------------------
+
+To close the connection, call the `Close` method of `SnowflakeDbConnection`.
+
+If you want to avoid blocking threads while the connection is closing, call the `CloseAsync` method instead.
+
```

Comment:
Is there an example of this that I can add here?

Reply:
Here is an example

```
CancellationTokenSource cancellationTokenSource  = new CancellationTokenSource();
// Close the connection
((SnowflakeDbConnection)conne).CloseAsync(cancellationTokenSource.Token);
```

Reply:
Thanks! I added this